v0.1/core/anatomicalEntity.schema.json

```diff
@@ -14,29 +20,40 @@
       "type": "string",
       "description": "Metadata node identifier.",
       "pattern": "sands/v0.1/core/anatomicalEntity/([A-Za-z0-9_-]+).json"
-    },
-    "nameOfLocation": {
-      "type": "string",
-      "description": "nameOfLocation"
-    },
+    },    
     "laterality": {
       "type": "array",
-      "description": "laterality",
+      "description": "The side of the body or bilateral organ that this anatomical entity relates to.",
       "items": {
         "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/CONTROLLED_TERM"
-      }
+      },
+      "maxItems": 2
+    },
+    "nameOfLocation": {
+      "type": "string",
+      "description": "The name of the anatomical entity."
     },
     "parentStructure": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/ANNATOMICAL_ENTITY",
-      "description": "parentStructure"
+      "type": "array",
```

Comment:
If the maximum item is 1 you don't have to define an array here. I would keep the json-schemas as simgle as possible. You can directly reference to the corresponding schema ("type": "object", "$ref": "..."). 
Let me know if you think otherwise.

Reply:
Yes, you're right. Most of them didn't have an object set as type. Does that mean it would be implicit that type = object when leaving it out and using $ref?

Reply:
For consistency I would always state the `"type"`, although in case `"$ref"` is used, the `"type": "object` is implied.